seqr/views/apis/project_api.py

```diff
@@ -197,6 +197,10 @@ def project_families(request, project_guid):
             JSONObject(paternalGuid='individual__father__guid', maternalGuid='individual__mother__guid'),
             filter=Q(individual__mother__isnull=False) | Q(individual__father__isnull=False), distinct=True,
         ),
+        phenotypePrioritizationTools=ArrayAgg(
+            'individual__phenotypeprioritization__tool', distinct=True,
+            filter=Q(individual__phenotypeprioritization__tool__isnull=False),
+        ),
```

Comment:
It looks like the only place this is used in the UI is to check `phenotypePrioritizationTools.length > 0`. If so,l it would be cleaner to define a boolean here, similar to `hasRequiredMetadata`
```
family_models = Family.objects.filter(project=project).annotate(
    pp_individual_count=Count('individual', filter=Q(individual__phenotypeprioritization__tool__isnull=False))
...
family_annotations = dict(
    hasPhenotypePrioritization=Case(When(pp_individual_count__gt=0, then=Value(True)), default=Value(False)),
...
```

Reply:
agree!